test_server.py

```diff
@@ -0,0 +1,38 @@
+from http.server import BaseHTTPRequestHandler, HTTPServer
```

Comment:
I'd recommend putting this into a `/contract-compiler` or `/py-server` directory. That way we can add a README.md or similar to that directory later on.